CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+
+> If you have a question, think you have discovered an issue or would like to propose a new feature or idea, then find/file an issue report **BEFORE** starting work to fix/implement it.
+
+## Open Development Workflow
+
+The Customer Architecture Team (CAT) is very active in this GitHub repository and carries out all the development work in the public. New issues, ideas or feature requests are filed in the repo and fixes and new implementations are implemented through pull requests that are linked to these issues. Code reviews can be followed as well to learn about the principles of the team and the development work.
+
+All the work of the CAT team is executed in the public to ensure a high degree of transparency for the community. In addition, it allows the community to more easily contribute to the projects because everyone is aware of new features that are commited or issues that are existing within the projects.
+
+Working in the public also ensures a high-quality bar and allows open discussions about the why and how of new developments.
+
+## Issue Reports
+
+We are encouraging you to select between a [Bug Report](#bug-report), a [Feature Request](#feature-request), a [Documentation Issue](#documentation-issue) and a [Vulnerability Report](#vulnerability-report) when opening a new Issue in this repository. Please provide all the required information for the respective issue reports and make sure you accept the [Code of Conduct](/CODE_OF_CONDUCT.md).
+
+Before submitting any of the types mentioned above, please search through the open and closed issues in this repository as well as through the [Known Issues](/docs/EnterpriseScaleAnalytics-KnownIssues.md). If a similar issue already exists, please upvote the existint issue comment(s) or leave a comment instead of opening a new issue. This helps us reduce the number issues and also simplifies the management on our side. Only if no existing item describes your issue report we encourage you to file a new one.
+
+### Bug Report
+
+Submit new bug reports using the "Bug Report" issue template. When submitting a bug report, try to be as specific as possible by including a detailed description of how to reproduce the issue and also share error messages and screenshots if possible. This helps us throubleshoot the bug and apply fixes more quickly.
+
+### Feature Request
+
+New feature requests and ideas should be submitted using the "Feature Request & Ideas" issue template. Please be specific about the "What?" and "Why?" by being specific about what the feature entails and what the benefits of including this feature will be.
+
+### Documentation Issue
+
+Documentation issues should be submitted using the "Documentation Issue" issue template. Please be specific about what document needs to be updated and why.
```

Comment:
Reframe, please.

"Documentation issues should be submitted using the "Documentation Issue" issue template. Please be specific about which document needs to be updated and why."